sql/core/src/main/scala/org/apache/spark/sql/execution/datasources/FileScanRDD.scala

```diff
@@ -53,12 +53,6 @@ case class PartitionedFile(
   }
 }
 
-/**
- * A collection of file blocks that should be read as a single task
- * (possibly from multiple partitioned directories).
- */
-case class FilePartition(index: Int, files: Seq[PartitionedFile]) extends RDDPartition
```

Comment:
Do we need to have those changes as @dongjoon-hyun suggested? Why not just change `Seq` to `Array` here?